src/Microsoft.ML.AutoML/AutoMLExperiment/IDatasetManager.cs

```diff
@@ -12,34 +15,83 @@ public interface IDatasetManager
     {
     }
 
-    internal interface ICrossValidateDatasetManager
+    public interface ICrossValidateDatasetManager
```

Comment:
Can you add some documentation for these since they are now public?

Reply:
Sure